.github/workflows/test-windows.yml

```
@@ -0,0 +1,57 @@
+name: Test Windows
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  test-windows:
+    name: Test Windows
+    runs-on: windows-latest
+    env:
+      _EXPERIMENTAL_DAGGER_RUNNER_HOST: ${{ secrets._EXPERIMENTAL_DAGGER_RUNNER_HOST }}
+      DAGGER_CLOUD_TOKEN: ${{ secrets.DAGGER_CLOUD_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Configure Git
+        run: |
+          git config --global user.email "test@example.com"
+          git config --global user.name "Test User"
+
+      - name: Install Dagger
+        shell: pwsh
+        run: |
+          choco install dagger -y
+          # refreshenv doesn't work in GitHub Actions, manually update PATH
+          $env:PATH = [System.Environment]::GetEnvironmentVariable("PATH", "Machine") + ";" + [System.Environment]::GetEnvironmentVariable("PATH", "User")
+          
+      - name: Verify Dagger installation
+        shell: pwsh
+        run: |
+          $env:PATH = [System.Environment]::GetEnvironmentVariable("PATH", "Machine") + ";" + [System.Environment]::GetEnvironmentVariable("PATH", "User")
+          dagger version
```

🐛 **Bug (Medium):** PATH update logic is duplicated and fragile. The manual PATH concatenation could fail if environment variables are null

```suggestion
          choco install dagger -y
          # refreshenv doesn't work in GitHub Actions, manually update PATH
          $machinePath = [System.Environment]::GetEnvironmentVariable("PATH", "Machine") ?? ""
          $userPath = [System.Environment]::GetEnvironmentVariable("PATH", "User") ?? ""
          $env:PATH = "$machinePath;$userPath".TrimStart(';')
          
      - name: Verify Dagger installation
        shell: pwsh
        run: |
          $machinePath = [System.Environment]::GetEnvironmentVariable("PATH", "Machine") ?? ""
          $userPath = [System.Environment]::GetEnvironmentVariable("PATH", "User") ?? ""
          $env:PATH = "$machinePath;$userPath".TrimStart(';')
```